### ISSUE TITLE
Evolution cookie fill be rejected soon...?

### ISSUE THREAD

**Issue by sashabeep** [evolution-cms/evolution#1466]
Вот чего прочитал в инструментах разработчика firefox
> Кука «evoa3f50g» вскоре будет отклоняться, поскольку её атрибут «sameSite» установлен в значение «none» или в недопустимое значение, и в ней отсутствует атрибут «secure». Чтобы узнать больше об атрибуте «sameSite», прочитайте https://developer.mozilla.org/docs/Web/HTTP/Cookies

**Comment by bossloper** [comment]
Yes I believe samesite=Lax needs adding? Solutions for this might be different for <=php7.2 
and >=php7.3.
Secure cookie is already supported and working if your site is on https and not http.

I did a quick test on Evo 1.4.x and replaced the setcookie line:
https://github.com/evolution-cms/evolution/blob/1.4.x/manager/includes/preload.functions.inc.php#L62

with:
```
setcookie(session_name(), session_id(), [
    'expires' => $cookieExpiration,
    'path' => $cookiePath,
    'domain' => $cookieDomain,
    'secure' => $secure,
    'httponly' => true,
    'samesite' => 'Lax'
]);
```
Work ok as I am on php7.4

**Comment by bossloper** [comment]
... further, if you run your site as secure https then you do not need to have 'samesite' explicitly set so the current Evo should be ok. Well that is how I understand it! :-)
